tony-examples/tony-in-gcp/README.md

```diff
@@ -258,11 +264,14 @@ gcloud beta dataproc autoscaling-policies import autoscale_tony --source=autosca
 export CLUSTER_NAME=tony-staging-1
 export DATAPROC_VERSION=1.3-deb9
 export ZONE=us-west1-a          # Update your zone/region accordingly
+export REGION=us-west1
 export BUCKET=tony-staging

```

Comment:
Can you update the BUCKET to ${BUCKET} as you did before?

Reply:
Done